dll/win32/shell32/lang/bg-BG.rc

```diff
@@ -94,7 +94,7 @@ CAPTION "Търсене на папка"
 FONT 8, "MS Shell Dlg"
 BEGIN
     DEFPUSHBUTTON "Добре", 1, 80, 176, 50, 12, BS_DEFPUSHBUTTON | WS_GROUP | WS_TABSTOP
-    PUSHBUTTON "Отказ", 2, 134, 176, 50, 12, WS_GROUP | WS_TABSTOP
+    PUSHBUTTON "Отказ", 2, 134, 176, 50, 12, WS_TABSTOP
```

Comment:
```suggestion
    PUSHBUTTON "Отказ", 2, 134, 176, 50, 12
```

Reply:
For the record: That is an unfair request @HBelusca ! He didn't touch that line. `And the whole file is full of that crap!` Let's better clean up the whole files instead **with a future PR**!

Reply:
OK then @Gathub22 can fix it only for the lines he originally had to modify to implement the intended fix for the PR.

Reply:
Ok, I will merge yours now. Please wait with opening another shell32 rc PR for 2 weeks, we do have at least 2 more PRs which are currently waiting for a merge, and which would collide with such a widespread diff in the RCs.
Act after merging them, one is a ro-RO specific one, the other touches all rcs. I will merge both, in the next 2 weeks most likely.